optuna/storages/in_memory.py

```diff
@@ -236,6 +247,31 @@ def set_trial_value(self, trial_id, value):
 
             self.trials[trial_id] = self.trials[trial_id]._replace(value=value)
 
+    def update_cache(self, trial_id):
```

Comment:
Please make this method private by adding `_` to the head of the name. I think this method is used only in this file.

Reply:
The method will be renamed in #611.